src/Field/Field.tsx

```diff
@@ -117,7 +117,9 @@ const ErrorBox = styled.span`
   font-size: 12px;
 `
 
-const Title = styled.h3`
+const Title = styled.label`
+  font-size: 16px;
+  line-height: 26px;
```

Comment:
are these coming from a specific design? @Jmuskett do you mind double check that these values are good with Direct Flow?

Reply:
Looking at Figma designs for direct flow it should follow this style
<img width="203" alt="Screenshot 2022-07-19 at 14 58 59" src="https://user-images.githubusercontent.com/1053476/179768787-e38193c8-b483-403a-8762-8aba2c63902c.png">

Reply:
I think we want it uniform across everything so it makes sense to use figma as the source of truth I think

Reply:
thanks! updated to reflect this.